modules/standard/Math.chpl

```diff
@@ -98,13 +98,24 @@ module Math {
   pragma "fn synchronization free"
   private extern proc fabs(x: real(64)): real(64);
 
-  private proc _logBasePow2Help(in val, baseLog2) {
-    var result = -1;
-    while (val != 0) {
-      val >>= baseLog2;
-      result += 1;
+  private inline proc _logBasePow2Help(in val, baseLog2) {
+    // These are used here to avoid including BitOps by default.
```

Comment:
Will a scoped `use BitOps` still include `BitOps` by default for user code?

Reply:
As far as some tests are concerned at least, it would. I'm not thinking about namespace pollution as much as which module inits are run and which modules are compiled (for compile time etc).

Reply:
Makes sense.